src/databricks/labs/ucx/workspace_access/scim.py

```diff
@@ -36,7 +36,7 @@ def get_crawler_tasks(self):
     # TODO remove after ES-892977 is fixed
     @retried(on=[DatabricksError])
     def _get_groups(self):
-        return list(self._ws.groups.list(attributes="id,displayName,roles,entitlements"))
+        return list(self._list_workspace_groups(scim_attributes="id,displayName,roles,entitlements"))
```

Comment:
```suggestion
        return list(self._list_workspace_groups("id,displayName,roles,entitlements"))
```
single keyed argument doesn't need to be explicit

Reply:
Reverted all `scim.py` changes because if only `members` attribute is handled differently, then `scim.py` is not impacted by the change.